scoring.js

```diff
@@ -43,39 +43,39 @@
         const rFum = receiving.fumbles * -3
         const kReturns = player.stats.return.kickreturn
         const pReturns = player.stats.return.puntreturn
-        const krYards = kReturns.yards / 10
+        const krYards = kReturns.yards / 15
```

Comment:
Good job getting everything in and working....that should be your first draft.  Then you should look into shortening the code.  Remember DRY don't repeat yourself.  LOTS of repetition here.  These can all be consolidated into smaller functions